### PR TITLE
fixed ontologies for general, georeference, hdmap and surface model

### PR DESCRIPTION
**Pull request by lenasauermann**
Added linked classes from shacl shape as ontology class. Also fixed a small error in georeference shacl.

I did it for general, georeference, hdmap and surface-model. I left scenario out as there are some linked classes, that only contain 1 attribute and therefore shouldn't be a sperate class.

### CODE REVIEW

**Comment by robertschubert** [comment]
@lenasauermann  please check the failed pipeline and assign functional reviewers as well.

**Comment by robertschubert** [comment]
> @lenasauermann please check the failed pipeline and assign functional reviewers as well.

I guess you need to merge main into this branch since there was a bugfix in the ci pipeline.

**Comment by robertschubert** [comment]
> > @lenasauermann please check the failed pipeline and assign functional reviewers as well.
> 
> I guess you need to merge main into this branch since there was a bugfix in the ci pipeline.

I merged main into this branch. Pipelines are running now.

File: general/general_ontology.ttl [hunk 1]
```diff
@@ -12,7 +12,31 @@ general: a owl:Ontology ;
     owl:versionInfo "0.1" .
 
 general:General a owl:Class ;
-    rdfs:label "class definition for General" ;
+    rdfs:label "Class definition for General" ;
     rdfs:comment "General attributes for all assets"@en ;
     rdfs:subClassOf gx:DataResource .
```
Comment (robertschubert) [inline general/general_ontology.ttl:18]: Please remove this line to have only 1 new line as separator

File: georeference/georeference_ontology.ttl [hunk 2]
```diff
@@ -16,3 +16,19 @@ georeference:Georeference a owl:Class ;
     rdfs:comment "Attributes for location and georeference information"@en ;
     rdfs:subClassOf gx:DataResource .
 
+
```
Comment (robertschubert) [inline georeference/georeference_ontology.ttl:19]: Please remove one line

File: hdmap/hdmap_ontology.ttl [hunk 3]
```diff
@@ -15,3 +15,24 @@ hdmap:HdMap a owl:Class ;
     rdfs:comment "Attributes for high definition maps"@en ;
     rdfs:subClassOf gx:DataResource .
 
+hdmap:Content a owl:Class ;
+    rdfs:label "class definition for Content" ;
+    rdfs:comment "Attributes for Content"@en .
+
```
Comment (robertschubert) [inline hdmap/hdmap_ontology.ttl:21]: Please remove one line

File: hdmap/hdmap_ontology.ttl [hunk 4]
```diff
@@ -15,3 +15,24 @@ hdmap:HdMap a owl:Class ;
...
+    rdfs:comment "Attributes for DataSource"@en .
+
+hdmap:Format a owl:Class ;
+    rdfs:label "class definition for Format" ;
+    rdfs:comment "Attributes for Format"@en .
+
+hdmap:Quality a owl:Class ;
+    rdfs:label "class definition for Quality" ;
+    rdfs:comment "Attributes for Quality"@en .
+
```
Comment (robertschubert) [inline hdmap/hdmap_ontology.ttl:34]: Please remove one line

File: surface-model/surface-model_ontology.ttl [hunk 5]
```diff
@@ -16,3 +16,20 @@ surface-model:SurfaceModel a owl:Class ;
     rdfs:comment "Attributes for surface-model"@en ;
     rdfs:subClassOf gx:DataResource .
 
+
```
Comment (robertschubert) [inline surface-model/surface-model_ontology.ttl:19]: Please remove one line

**Review by robertschubert — CHANGES_REQUESTED** [review]

**Review by robertschubert — APPROVED** [review]
all fine for me now :)

**Comment by robertschubert** [comment]
@lenasauermann  please assign a second reviewer

**Review by jtdemer — APPROVED** [review]
fine for me as well, thx lena and robert

**Comment by robertschubert** [comment]
2 positive reviews, I will merge it.

**Pull request merged**